javascript/core/objects/object-part-2.md

```diff
@@ -20,7 +20,9 @@ tags:
   - introduction
 
 aspects:
-  - introduction
+  - deep
```

Comment:
Not `deep` or `obscura`, you're going to need this to do a lot of `introduction` level content we serve.